core/emu.py

```diff
@@ -1,13 +1,15 @@
 #!/usr/bin/env python3
+from __future__ import annotations
```

Comment:
Is this required?

I don't see any annotation where we'd need postponed evaluation, or I am wrong here?

Reply:
I am using "union pipe" two times  - `str | Path` - and without the `future` import this fails at runtime - as we have default `3.9` python, it works natively from `3.10`

Reply:
thanks, TIL